doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1235,6 +1235,10 @@ Other libraries
 
   * Added the :c:func:`date_time_now_local` function to the API.
 
+* :ref:`dk_buttons_and_leds_readme` library:
+
+  * Added experimental no interrupts mode for button handling, which can be enabled with the :kconfig:option:`CONFIG_DK_LIBRARY_BUTTON_NO_ISR` Kconfig option.
```

Comment:
Maybe we could mention here that it increases power consumption and was added specifically for nRF54L PDK v0.2.1 (and rather should not be used for other boards)? @kapi-no  what do you think?

Reply:
Added only info that this is workaround for nrf54l pdk rev 0.2.x issues, more details can be checked in the kconfig help